autoload/writable_search/parser.vim

```diff
@@ -101,6 +111,23 @@ function! s:BuildProxies(grouped_lines)
   return proxies
 endfunction
 
+function! s:BuildProxiesFromQuickfix()
+  let proxies = []
+  let quickfix_list = getqflist()
+
+  for location in quickfix_list
+    let current_proxy = writable_search#proxy#New(location.bufnr)
```

Comment:
Ah, no, wait, my bad, you're just using one per patch. Completely forgot the implementation of my own plugin :D.